### PR TITLE
[CCV-1373] Improve GCP documentation 

### PR DESCRIPTION
**Pull request by klara-kr**
### What does this PR do? What is the motivation?

We want to improve the documentation to make it easier for customers to set up GCP with CCM. This PR aims to restructure sentences and make them easier to follow and match the vocabulary/steps of the official GCP documentation better.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/cloud_cost_management/google_cloud.md [hunk 1]
```diff
@@ -26,63 +26,64 @@ Cloud Cost for Google Cloud is in public beta
...
   * `bigquery.transfers.get`
   * `bigquery.transfers.update`
 
   **Note:** This can be a custom role, or you can use the existing Google Cloud role `roles/bigquery.admin`.
 
-#### **Configure required export BigQuery dataset access**:
-[Add the service account as a principal on the export BigQuery dataset resource][8] and assign a role with the following permissions:
+#### Configure export BigQuery dataset access
+[Add the service account as a principal on the export BigQuery dataset resource][8]:
+1. In the Explorer pane on the BigQuery page expand your project and select the export BigQuery dataset.
```
Comment (crousseaux) [inline content/en/cloud_cost_management/google_cloud.md:83]:
```suggestion
1. In the Explorer pane on the BigQuery page, expand your project and select the export BigQuery dataset.
```

File: content/en/cloud_cost_management/google_cloud.md [hunk 2]
```diff
@@ -95,17 +96,37 @@ The following APIs and permissions enable Datadog to access your Detailed Usage
 
   **Note:** This can be a custom role, or you can use the existing Google Cloud role `roles/bigquery.dataEditor`.
 
-#### (Optional) Configure required cross-project service authorization:
+### Create or select a Google Cloud Storage bucket
+Use an existing Google Cloud Storage bucket or create a new one.
+Data is extracted regularly from your Detailed Usage Cost BigQuery dataset to the selected bucket, where it will be prefixed with `datadog_cloud_cost_detailed_usage_export`.
```
Comment (crousseaux) [inline content/en/cloud_cost_management/google_cloud.md:101]:
```suggestion
Data is extracted regularly from your Detailed Usage Cost BigQuery dataset to the selected bucket and prefixed with `datadog_cloud_cost_detailed_usage_export`.
```

File: content/en/cloud_cost_management/google_cloud.md [hunk 3]
```diff
@@ -95,17 +96,37 @@ The following APIs and permissions enable Datadog to access your Detailed Usage
...
+  **Note:** This can be a custom role, or you can use the existing Google Cloud roles `roles/storage.legacyObjectReader` and `roles/storage.legacyBucketWriter`.
+
+### (Optional) Configure cross-project service authorization:
 If your integrated Service Account exists in a different Google Cloud Platform project than your billing export dataset, you need to [grant cross-project service account authorization][10]:
 
-[Trigger service agent creation][11] using the following values:
+1. Trigger the service agent creation by following the [official documentation][11] using the following values:
   * ENDPOINT: `bigquerydatatransfer.googleapis.com`
   * RESOURCE_TYPE: `project`
   * RESOURCE_ID: export dataset project
```
Comment (crousseaux) [inline content/en/cloud_cost_management/google_cloud.md:125]: Debating if we should indent these 3 bullet points to improve readability 🧐 

Screenshot 2024-02-09 at 13 56 08

File: content/en/cloud_cost_management/google_cloud.md [hunk 4]
```diff
@@ -95,17 +96,37 @@ The following APIs and permissions enable Datadog to access your Detailed Usage
...
 
-  This creates a new service agent that looks like `service-<billing project number>@gcp-sa-bigquerydatatransfer.iam.gserviceaccount.com`.
-
-Add the BigQuery Data Transfer Service Account role created by the trigger as a principal on your Service Account resource with the `roles/iam.serviceAccountTokenCreator` role.
+    This creates a new service agent that looks like `service-<billing project number>@gcp-sa-bigquerydatatransfer.iam.gserviceaccount.com`.
+2. Add the BigQuery Data Transfer Service Account role created by the trigger as a principal on your Service Account
+3. Assign it the `roles/iam.serviceAccountTokenCreator` role.
 
 ### Configure Cloud Costs
 Navigate to [Setup & Configuration][3] and follow the steps provided in-app.
```
Comment (crousseaux) [inline content/en/cloud_cost_management/google_cloud.md:132]:
```suggestion
Continue to follow the steps indicated in  [Setup & Configuration][3].
```

**Review by crousseaux — APPROVED** [review]
ltgm, thanks for addressing feedback

File: content/en/cloud_cost_management/google_cloud.md [hunk 5]
```diff
@@ -26,61 +26,59 @@ Cloud Cost for Google Cloud is in public beta
...
-   * `storage.objects.list`
+  **Note:** BigQuery Data Transfer API needs to be enabled on the Google Project that contains the service account.
 
-  **Note:** This can be a custom role, or you can use the existing Google Cloud roles `roles/storage.legacyObjectReader` and `roles/storage.legacyBucketWriter`.
 
-#### **Configure required export project access**:
-[Add the service account as a principal on the export dataset project resource][7] and assign a role with the following permissions:
+#### Configure export project access
+[Add the service account as a principal on the export dataset project resource][7]:
+1.  Navigate to the IAM page in the Google Cloud console and select the export dataset project.
```
Comment (aliciascott) [inline content/en/cloud_cost_management/google_cloud.md:67]:
```suggestion
1. Navigate to the IAM page in the Google Cloud console and select the export dataset project.
```

File: content/en/cloud_cost_management/google_cloud.md [hunk 6]
```diff
@@ -26,61 +26,59 @@ Cloud Cost for Google Cloud is in public beta
...
+ 3. Document the `Billing Account ID` for the billing account where the export was configured, as well as the export `Project ID` and `Dataset Name`.
 
-### Provide service account necessary permissions
-The following APIs and permissions enable Datadog to access your Detailed Usage billing export data and extract it in a useful format. This dumps data from BigQuery, where Google Cloud exports it, to your specified GCS bucket through a scheduled BigQuery query. The data is exported in a useful manner in your cloud storage bucket where it can then be processed by Datadog.
+#### Enable Google Service APIs
+The following permissions allow Datadog to access and transfer the billing export into the storage bucket using a scheduled BigQuery query.
 
-#### **Enable necessary Google Service APIs**:
-[Enable BigQuery and BigQuery Data Transfer Service APIs][5] if not already enabled.
+- Enable the [BigQuery API][5]
```
Comment (aliciascott) [inline content/en/cloud_cost_management/google_cloud.md:53]:
```suggestion
- Enable the [BigQuery API][5].
```

File: content/en/cloud_cost_management/google_cloud.md [hunk 7]
```diff
@@ -26,61 +26,59 @@ Cloud Cost for Google Cloud is in public beta
...
+  1. In the Google Cloud console, go to the project selector page and select your Google Cloud project.
+  2. Enable billing on your project for all transfers.
 
-**Note:** BigQuery Data Transfer API needs to be enabled on the Google Project that contains the service account.
+- Enable the [BigQuery Data Transfer Service][5]
```
Comment (aliciascott) [inline content/en/cloud_cost_management/google_cloud.md:57]:
```suggestion
- Enable the [BigQuery Data Transfer Service][5].
```

File: content/en/cloud_cost_management/google_cloud.md [hunk 8]
```diff
@@ -26,61 +26,59 @@ Cloud Cost for Google Cloud is in public beta
...
   * `bigquery.transfers.update`
 
   **Note:** This can be a custom role, or you can use the existing Google Cloud role `roles/bigquery.admin`.
 
-#### **Configure required export BigQuery dataset access**:
-[Add the service account as a principal on the export BigQuery dataset resource][8] and assign a role with the following permissions:
+#### Configure export BigQuery dataset access
+[Add the service account as a principal on the export BigQuery dataset resource][8]:
+1. In the Explorer pane on the BigQuery page, expand your project and select the export BigQuery dataset.
+2. Click 'Sharing > Permissions' and then 'Add principal'.
```
Comment (aliciascott) [inline content/en/cloud_cost_management/google_cloud.md:79]:
```suggestion
2. Click **Sharing > Permissions** and then **add principal**.
```

File: content/en/cloud_cost_management/google_cloud.md [hunk 9]
```diff
@@ -26,61 +26,59 @@ Cloud Cost for Google Cloud is in public beta
...
+3. In the New principals field, enter the service account.
```
Comment (aliciascott) [inline content/en/cloud_cost_management/google_cloud.md:80]:
```suggestion
3. In the new principals field, enter the service account.
```

File: content/en/cloud_cost_management/google_cloud.md [hunk 10]
```diff
@@ -26,61 +26,59 @@ Cloud Cost for Google Cloud is in public beta
...
+4. Using the Select a role list, assign a role with the following permissions:
```
Comment (aliciascott) [inline content/en/cloud_cost_management/google_cloud.md:81]:
```suggestion
4. Using the select a role list, assign a role with the following permissions:
```

File: content/en/cloud_cost_management/google_cloud.md [hunk 11]
```diff
@@ -93,20 +91,42 @@ The following APIs and permissions enable Datadog to access your Detailed Usage
...
+### Create or select a Google Cloud Storage bucket
+Use an existing Google Cloud Storage bucket or create a new one.
+Data is extracted regularly from your Detailed Usage Cost BigQuery dataset to the selected bucket and prefixed with `datadog_cloud_cost_detailed_usage_export`.
+
+**Note:** The bucket [must be co-located][9] with the BigQuery export dataset.
+
+#### Configure bucket access
+[Add the service account as a principal on the GCS bucket resource][6]:
+1. Navigate to the Cloud Storage Buckets page in the Google Cloud console and select your bucket.
+2. Select the Permissions tab and click the 'Grant access' button.
```
Comment (aliciascott) [inline content/en/cloud_cost_management/google_cloud.md:103]:
```suggestion
2. Select the permissions tab and click the **grant access** button.
```

File: content/en/cloud_cost_management/google_cloud.md [hunk 12]
```diff
@@ -93,20 +91,42 @@ The following APIs and permissions enable Datadog to access your Detailed Usage
...
-#### (Optional) Configure required cross-project service authorization:
+### Create or select a Google Cloud Storage bucket
+Use an existing Google Cloud Storage bucket or create a new one.
+Data is extracted regularly from your Detailed Usage Cost BigQuery dataset to the selected bucket and prefixed with `datadog_cloud_cost_detailed_usage_export`.
+
+**Note:** The bucket [must be co-located][9] with the BigQuery export dataset.
+
+#### Configure bucket access
+[Add the service account as a principal on the GCS bucket resource][6]:
+1. Navigate to the Cloud Storage Buckets page in the Google Cloud console and select your bucket.
```
Comment (aliciascott) [inline content/en/cloud_cost_management/google_cloud.md:102]:
```suggestion
1. Navigate to the Cloud Storage Buckets page in the Google Cloud console, and select your bucket.
```

File: content/en/cloud_cost_management/google_cloud.md [hunk 13]
```diff
@@ -93,20 +91,42 @@ The following APIs and permissions enable Datadog to access your Detailed Usage
...
 If your integrated Service Account exists in a different Google Cloud Platform project than your billing export dataset, you need to [grant cross-project service account authorization][10]:
 
-[Trigger service agent creation][11] using the following values:
-  * ENDPOINT: `bigquerydatatransfer.googleapis.com`
-  * RESOURCE_TYPE: `project`
-  * RESOURCE_ID: export dataset project
+1. Trigger the service agent creation by following the [official documentation][11] using the following values:
+   * ENDPOINT: `bigquerydatatransfer.googleapis.com`
+   * RESOURCE_TYPE: `project`
+   * RESOURCE_ID: export dataset project
```
Comment (aliciascott) [inline content/en/cloud_cost_management/google_cloud.md:120]:
```suggestion
   * RESOURCE_ID: export dataset project</br>
```

**Review by aliciascott — CHANGES_REQUESTED** [review]
Hi @klara-kr looks great, just a few minor changes for punctuation and spacing, please request re review once the changes are updated and I'll get it merged :)

**Review by aliciascott — APPROVED** [review]
looks good!

**Pull request merged**